docs/lecture1.md

```diff
@@ -3,6 +3,8 @@ id: lecture1
 title: Lecture 1 - ES6 Basics
 ---
 
+[Lecture Slides](https://docs.google.com/presentation/d/1y_7EWaEBllqTSQOZw_43m02-wh49DatEh0OVSDJkf4A/edit?usp=sharing)
```

Comment:
Can you make the link viewable by anyone?

Reply:
Updated